### PR TITLE
Remove the expectation on page content for deleting a draft

### PR DESCRIPTION
**Pull request by cbaines**
Just check the response code, which is what I think is really being
tested here.

### CODE REVIEW

**Comment by kevindew** [comment]
This seems sensible. Do you know what comes up instead of a variation on "Not found"?

The thing that's a bit of a bummer is that this removes the expectation from the test though. So it might be that it's best to change it to something along the lines of:

```
visit(@url)
expect(response.status_code).to be 404
```

It does leave me feeling a bit in a rock and a hard place though where it feels a little wrong for the test to not have an expectation and also wrong to re-test something we just waited for :-/

**Comment by cbaines** [comment]
> This seems sensible. Do you know what comes up instead of a variation on "Not found"?

Yep `expected to find text "Not found" in "404 page not found"`.

> The thing that's a bit of a bummer is that this removes the expectation from the test though. So it might be that it's best to change it to something along the lines of:
> 
> visit(@url)
> expect(response.status_code).to be 404
> 
> It does leave me feeling a bit in a rock and a hard place though where it feels a little wrong for the test to not have an expectation and also wrong to re-test something we just waited for :-/

Hmm, if the issue is not having something that says expectation, we could alias `reload_url_until_status_code` to say `expect_status_code_eventually`?

**Comment by kevindew** [comment]
> Hmm, if the issue is not having something that says expectation, we could alias reload_url_until_status_code to say expect_status_code_eventually?

Yeah that sounds like a good idea 👍

File: spec/support/retry_helpers.rb [hunk 1]
```diff
@@ -66,6 +66,8 @@ def reload_url_until_status_code(
     end
   end
 
+  alias reload_url_until_status_code expect_status_code_eventually
```
Comment (kevindew) [inline spec/support/retry_helpers.rb]: I think these have to be the other way round

Comment (cbaines) [reply spec/support/retry_helpers.rb]: Good spot, I've switched the order now.

**Comment by cbaines** [comment]
I've rebased this atop https://github.com/alphagov/publishing-e2e-tests/pull/75 , just to see if I can get it to pass in CI... assuming https://github.com/alphagov/publishing-e2e-tests/pull/75 gets merged, I'll rebase it again on master prior to merging.

**Pull request merged**